From: Branden Robinson <branden@debian.org>

Fix system.twmrc to actually be useful and tabbed right, as well as add
the Debian menu.

Index: twm/src/system.twmrc
===================================================================
--- twm.orig/src/system.twmrc
+++ twm/src/system.twmrc
@@ -48,7 +48,8 @@ Function "move-or-iconify" { f.move f.de
 #
 # Set some useful bindings.  Sort of uwm-ish, sort of simple-button-ish
 #
-Button1 = : root : f.menu "defops"
+Button1 = : root : f.menu "main"
+Button2 = : root : f.menu "windowops"
 
 Button1 = m : window|icon : f.function "move-or-lower"
 Button2 = m : window|icon : f.iconify
@@ -66,25 +67,39 @@ Button2 = : iconmgr : f.iconify
 #
 # And a menus with the usual things
 #
-menu "defops"
+menu "main"
 {
-"Twm"	f.title
-"Iconify"	f.iconify
-"Resize"	f.resize
-"Move"		f.move
-"Raise"		f.raise
-"Lower"		f.lower
-""		f.nop
-"Focus"		f.focus
-"Unfocus"	f.unfocus
-"Show Iconmgr"	f.showiconmgr
-"Hide Iconmgr"	f.hideiconmgr
-""		f.nop
-"Xterm"		f.exec "exec xterm &"
-""		f.nop
-"Kill"		f.destroy
-"Delete"	f.delete
-""		f.nop
-"Restart"	f.restart
-"Exit"		f.quit
+		"Main Menu"		f.title
+		"Debian"		f.menu "/Debian"
+		""			f.nop
+		"Show Icon Manager"	f.showiconmgr
+		"Hide Icon Manager"	f.hideiconmgr
+		""			f.nop
+		"Exit"			f.menu "Quit-Verify"
 }
+
+menu "Quit-Verify"
+{
+		"Really quit twm?"	f.title
+		"No, restart twm"	f.restart
+		"Yes, really quit"	f.quit
+}
+
+menu "windowops"
+{
+		"Window Ops"		f.title
+		""			f.nop
+		"Iconify"		f.iconify
+		"Resize"		f.resize
+		"Move"			f.move
+		"Raise"			f.raise
+		"Lower"			f.lower
+		""			f.nop
+		"Focus"			f.focus
+		"Unfocus"		f.unfocus
+		""			f.nop
+		"Delete"		f.delete
+		"Destroy"		f.destroy
+}
+
+include-menu-defs
Index: twm/src/Makefile.am
===================================================================
--- twm.orig/src/Makefile.am
+++ twm/src/Makefile.am
@@ -24,13 +24,13 @@ AM_YFLAGS = -d
 
 bin_PROGRAMS = twm
 
-rcdir = ${datadir}/X11/twm
+rcdir = ${sysconfdir}/X11/twm
 dist_rc_DATA = system.twmrc
 
 AM_CPPFLAGS =							\
 	-DXVENDORNAME=\"The\ X.Org\ Foundation\"		\
 	-DXORG_RELEASE=\"Release\ $(VERSION)\"			\
-	-DSYSTEM_INIT_FILE=\"${datadir}/X11/twm/system.twmrc\"
+	-DSYSTEM_INIT_FILE=\"$(rcdir)/system.twmrc\"
 
 AM_CFLAGS = $(TWM_CFLAGS)
 twm_LDADD = $(TWM_LIBS)
